app/views/schools/advice/_advice_page.html.erb

```diff
@@ -14,6 +12,7 @@
     <%= render 'schools/advice/nav', school: school, advice_pages: advice_pages %>
   </div>
   <div class="col-md-10 advice-page-tabs">
+    <h1><%= "#{advice_page.key.humanize} analysis and advice" %></h1>
```

Comment:
does this string need extracting out so it can be translated?

Reply:
Good point! - I'll make it translatable